cfgov/jinja2/v1/_includes/organisms/snippet-list.html

```diff
@@ -55,7 +55,7 @@
     } ) }}
 
     {% set snippets = get_snippets(value.snippet_type) %}
-    {% set filtered_snippets = snippets.objects.filter_by_tags(value.tags) %}
+    {% set filtered_snippets = snippets.objects.filter_by_tags(value.tags) if snippets.objects.filter_by_tags else snippets.objects.all() %}
```

Comment:
Why yes, it turns out that Jinja statements CAN be broken onto multiple lines! Fixed in  	d7602b5.